src/main.qml

```diff
@@ -1614,7 +1614,7 @@ ApplicationWindow {
             }
         }
 
-        imageWriter.setHWFilterList(hwmodel.tags, inclusive)
+        imageWriter.setHWFilterList(hwmodel.tags, inclusive, hwmodel.name.toLowerCase().includes("zero"))
```

Comment:
This makes me nervous, but I can see the motive.

Reply:
You can rest now sure,. that this check isn't in-use anymore. Has been replaced by new HW Supported Features list.